src/components/filters/AttributeFilter/AttributeDropdown.tsx

```diff
@@ -297,7 +300,12 @@ export class AttributeDropdownWrapped extends React.PureComponent<
     };
 
     private onSearch = (searchString: string) => {
-        this.setState({ searchString });
+        const { attributeDisplayForm } = this.props;
+        this.setState({
+            searchString,
+            isListReady: false,
+        });
+        this.setupDataSource(attributeDisplayForm.meta.uri);
```

Comment:
There are way easier ways to get attribute values. DataSource is too heavy for a single request to one URI.

What about sdk.md.getValidElements?

Reply:
Yeah, I know this component deserves a lot more ❤️, but I don't really have time to dig deeper 😢 Take it or leave it.